api/src/test/java/org/openmrs/api/db/hibernate/HibernateOrderDAOTest.java

```diff
@@ -65,4 +74,82 @@ public void saveOrderGroup_shouldSaveOrderGroup() {
 		}
 		
 	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupsByEncounter(Encounter)}
+	 * @throws Exception
+	 */
+	@Test(expected = APIException.class)
+	public void getOrderGroupsByEncounter_shouldFailGivenNullEncounter() {
+		dao.getOrderGroupsByEncounter(null);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupsByPatient(Patient)}
+	 * @throws Exception
+	 */
+	@Test(expected = APIException.class)
+	public void getOrderGroupsByPatient_shouldFailGivenNullPatient() {
+		dao.getOrderGroupsByPatient(null);
+	}
+	
+	/**
+	 * @see {@link HibernateOrderDAO#getOrderGroupsByEncounter(Encounter)}
+	 * @throws Exception
+	 */
+	@Test
+	public void getOrderGroupsByEncounter_shouldGetOrderGroupsFromAnEncounter() {
+		Encounter existingEncounter1 = Context.getEncounterService().getEncounter(4);
```

Comment:
Is there any reason why you are appending `1` to the variable names?

Reply:
going to fix it